v2/client.go

```diff
@@ -18,10 +18,11 @@ import (
 )
 
 const (
-	// XBrokerAPIVersion is the header for the Open Service Broker API
-	// version.
+	// APIVersionHeader specifies use of the XBrokerAPIVersion header for the Open
```

Comment:
is the header value associated with the version of the Open...

Reply:
The line has to start with APIVersionHeader--I'm not sure how best to say what was already there but starting with that.